### PR TITLE
i18n(ja): Update Japanese `framework-components.md`, `troubleshooting.md`

### PR DESCRIPTION
**Pull request by kyosuke**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Updated Japanese `framework-components.md` to match the English update
    - https://github.com/withastro/docs/commit/6f131d3a68a2b3446e9acef763e52977ae27fa16
    - https://github.com/withastro/docs/commit/011472dec09a8ea4a3b0cb2fe15e11cf7dc6e228
    - https://github.com/withastro/docs/commit/c81e16af9a287b53accd0945cd7c08f6ecac591e
    - https://github.com/withastro/docs/commit/8412b41cebbfd7f116b9ba015028c9ee24494bf9
    - https://github.com/withastro/docs/commit/c72ed8bdd3fa5631f1639f6529dfdb595ae2b456
    - https://github.com/withastro/docs/commit/1c900121aa0c5b91761c4cd99997a692312510a6
    - https://github.com/withastro/docs/commit/b3039c2310e524c17c56f6cac421fc478513bf4e
    - https://github.com/withastro/docs/commit/d649ada446c9bc30b665cc6dc964ab68fd8ef723
    - https://github.com/withastro/docs/commit/162aecc935dbd60a9ccd517cb9159f1c7b39ebba
    - https://github.com/withastro/docs/commit/32ce58a0287c8f573b65fb2b5fe39a3a3acae76f
    - https://github.com/withastro/docs/commit/44d67ed59cb292306a23c18aad61b40a4f8b609c
    - Removed unnecessary spaces.
    - Made it easier to read as Japanese.
    - Matched the differences with the latest English version.
- Updated Japanese `troubleshooting.md` to match the English update
    - https://github.com/withastro/docs/commit/1900d02f00450970a6b3099883abdc0b529d5ad2
    - https://github.com/withastro/docs/commit/09a88b40693d0a003835902bde028650915c9d72
    - Fixed a link to framework components page

### CODE REVIEW

File: src/pages/ja/core-concepts/framework-components.md [hunk 1]
```diff
@@ -118,42 +110,102 @@ import MyVueComponent from '../components/MyVueComponent.vue';
...
+## フレームワークコンポーネントに子コンポーネントを渡す
+
+Astroコンポーネントでは、フレームワークコンポーネントに子コンポーネントを**渡せます**。各フレームワークは、これらの子コンポーネントを参照するための固有のパターンがあります。React、Preact、Solidは`children`という特別なプロパティを使用し、SvelteとVueは`<slot />`という要素を使用します。
+
+```astro title="src/pages/component-children.astro" {5}
+---
+import MyReactSidebar from '../components/MyReactSidebar.jsx';
+---
+<MyReactSidebar>
+  <p>Here is a sidebar with some text and a button.</p>
```
Comment (Yan-Thomas) [inline src/pages/ja/core-concepts/framework-components.md:127]: Pointing out this, I think you forgot to translate this part.

**Review by Yan-Thomas — COMMENTED** [review]
Good job @kyosuke, I added a comment pointing out something you have probably missed. And I think these pages went under some changes recently, could you please update them? I'll review and merge it ASAP! Thank you :purple_heart:

**Comment by kyosuke** [comment]
Thanks @Yan-Thomas, I've updated it to match the commits added to the latest English version. I have also fixed the omissions you pointed out.

**Comment by Yan-Thomas** [comment]
LGTM!

**Pull request merged**